packages/aragon-wrapper/src/index.js

```diff
@@ -772,6 +774,29 @@ export default class Aragon {
     return this.accounts.pipe(first()).toPromise()
   }
 
+  /**
+   * Allows apps to sign arbitrary data via a RPC call
+   *
+   * @param {Array<Object>} params An object containing the address of the app and the data to be signed
+   * @return {Promise<string>} signature hash
+   */
+  signMessage (params) {
+    const { fromAddress, message } = params
+
+    return new Promise((resolve, reject) => {
+      this.signatures.next({
+        from: fromAddress,
```

Comment:
again, wondering if we should rename this, or also include the `account` that will sign the message